doc/dialects/elasticsearch_sql_user_guide.md

```diff
@@ -1,129 +1,3 @@
-
-For referencing nested fields directly in an ElasticSearch data source, the dot(.) is used as identifier separator, so if you want to refer to the first name of the author, you would do it as `author.first_name`, as is in the following example query executed directly in the ElasticSearch database:
-
-```sql
-SELECT "author.first_name" FROM "book";
-```
-
-In Exasol however the dot(.) can not be used inside quoted identifiers, which means that `author.first_name` is not allowed. On the other hand, as explained above, you should always use quoted identifiers given the case sensitive nature of ElasticSearch.
-
-To overcome this situation, every nested field from ElasticSearch is automatically converted to the virtual schema in the Exasol database using the slash(/) as identifier separator. Hence, `author.first_name` becomes `author/first_name`, so the previous example query becomes:
-
-```sql
-SELECT "author/first_name" FROM "<elastic_search_virtual_schema_name>"."book";
-```
-
-### ElasticSearch Mapping
-
-In ElasticSearch, [mapping](https://www.elastic.co/guide/en/elasticsearch/reference/current/mapping.html) is the process of defining how a document, and the fields it contains, are stored and indexed. Among other things, the mapping of documents includes the definition of the fields that can be used for performing queries over those documents, and more over these fields are going to be available to perform queries from the Exasol database using the VirtualSchema.
-
-ElasticSearch allows [dynamic mapping](https://www.elastic.co/guide/en/elasticsearch/reference/current/dynamic-mapping.html), which means that fields and mapping types do not need to be defined before being used, and the new field names will be added automatically, just by indexing a document.
-
-If the new fields are added after creating VirtualSchema, those fields will not be available, as the VirtualSchema is not up to date at this point, but it can be updated by executing the following command:
-
-```sql
-ALTER VIRTUAL SCHEMA "<elastic_search_virtual_schema_name>" REFRESH;
-```
-
-Given the previously described situation, we recommend to not use the dynamic mapping feature of ElasticSearch, and instead define the mappings explicitly.
-
-### Known issues
-
-At the moment for querying all the contents of a table in the virtual schema it is not possible to use the `*` wild card, as in:
-
-```sql
-SELECT * FROM "<elastic_search_virtual_schema_name>"."<table_name>";
-```
-
-Instead the columns to query needs to be explicitly specified, as in:
-
-```sql
-SELECT "<column_name_1>","<column_name_2>",...,"<column_name_n>" FROM "<elastic_search_virtual_schema_name>"."<table_name>";
-```
-
-A new feature to avoid this issue will be added in the future.
+The ElasticSearchSQL dialect guide can be found at (https://github.com/exasol/elasticsearch-virtual-schema/blob/main/doc/user_guide/elasticsearch_sql_user_guide.md).
```

Comment:
This link doesn't work

Reply:
I think the path is https://github.com/exasol/elasticsearch-virtual-schema/blob/main/doc/dialects/elasticsearch_sql_user_guide.md